### ISSUE TITLE
HOW I CAN INSTALL THIS PERFECT TOOL IN BLACKARCH ?

### ISSUE THREAD

**Issue by haytam776** [Chudry/Xerror#3]
HOW I CAN INSTALL THIS PERFECT TOOL IN BLACKARCH ?

**Comment by abdulmyid** [comment]
readme wont help :)

**Comment by meetgyn** [comment]
How to use this porject:
1.Activate virtual enviroment by following command
souce env/bin/activate
2. Start redis server
service redis-server start
3. start python srver
1. cd xerror 2. python mana.py runserver
4. start celery server
1. cd xerror
2. celery -A xerror worker -l info
5. start msfrpc server
msfrpcd -P 123 -S -a 127.0.0.1
6. start openvas server for default set OMP server credientials to admin@admin 127.0.0.1 9392



**your readme is very confusing.
Could you help us? Do I need to have openvas installed? Celery?
this line got really confused 1. cd xerror 2. python mana.py runserver. Cade o mana.py?**

**Comment by GraylockInc** [comment]
Yeah maybe you should add an autorun script because the start process doesn't work for me as well.

For example the missing mana.py command in the readme, there is manage.py but combining runserver command does not work.

**Comment by adilshafiq1512** [comment]
> How to use this porject:
> 1.Activate virtual enviroment by following command
> souce env/bin/activate
> 2. Start redis server
> service redis-server start
> 3. start python srver
> 
> 1. cd xerror 2. python mana.py runserver
> 2. start celery server
> 3. cd xerror
> 4. celery -A xerror worker -l info
> 5. start msfrpc server
>    msfrpcd -P 123 -S -a 127.0.0.1
> 6. start openvas server for default set OMP server credientials to admin@admin 127.0.0.1 9392
> 
> **your readme is very confusing. Could you help us? Do I need to have openvas installed? Celery? this line got really confused 1. cd xerror 2. python mana.py runserver. Cade o mana.py?**

Yes you have to install openvas. Personally i didn't tried on this project just seen a link in a fb and came here and tried to analyse whole project. He just put together all things like scanning then suitable attack vector for a target /exploits/vulnerabilities etc then exploitation then post etc etc. Follow the instructions you will get your universal easy to use GUI pentesting  broswer based application.  I will try to work on this project further when i get time little bit from. Other main projects

**Comment by Knoa6** [comment]
pip install -r requirements.txt should install openvas. The issue for me was it being based on an unsupported python version that requires pyenv   but the all the python2.7 packages like Django conflict with the modern versions used by python3.. i just dont get why python3 wasn't used here and the whole nature of this just seems suss like there is an ulterior motive but of course I'm prolly just being paranoid

**Comment by Ammarusatx** [comment]
is thing even real ,, is there a way to install it

**Comment by hotepbubba** [comment]
this thing is completely broken
